ignite/services/chain/init.go

```diff
@@ -229,10 +229,6 @@ func (c *Chain) IsInitialized() (bool, error) {
 	if _, err := os.Stat(gentxDir); os.IsNotExist(err) {
 		return false, nil
 	}
-	if err != nil {
```

Comment:
why was it removed?

Reply:
ah this is a new check, for "nilness" in that case above, err is always nil, because if there is an error, we will return false, nil